tests/backup/backup_sse_test.go

```diff
@@ -144,11 +146,11 @@ var _ = Describe("{CreateBackupAndRestoreForAllCombinationsOfSSES3AndDenyPolicy}
 				cloudCredUidForBlWithoutSse = uuid.New()
 				cloudCredUidList = append(cloudCredUidList, cloudCredUID)
 				backupLocationUidWithoutSse = uuid.New()
-				credNameForBlWithoutSse = fmt.Sprintf("autogenerated-cred--%s-%v", RandomString(randomStringLength), time.Now().Unix())
+				credNameForBlWithoutSse = fmt.Sprintf("autogenerated-cred--%s-%v", RandomString(randomStringLength), RandomString(5))
```

Comment:
I feel there is no need to add RandomString(5) cause we already have RandomString(randomStringLength).

Reply:
corrected